lms/services/youtube.py

```diff
@@ -0,0 +1,23 @@
+class YoutubeService:
+    """An interface for dealing with Youtube API."""
+
+    def __init__(self, enabled: bool):
+        """
+        Initialise the Youtube service.
+
+        :param enabled: Whether Youtube is enabled on this instance
+        """
+        self._enabled = enabled
+
+    @property
+    def enabled(self) -> bool:
+        """Get whether this instance is configured for Youtube."""
+
+        return self._enabled
```

Comment:
A bit silly just for this but:
- It aligns with other services with the same patter (eg Jstor)
- This will get more complex as we add the youtube API etc.